welcoming.txt

```diff
@@ -19,4 +19,15 @@
   . Lá encontraste um server de developers e agora queres que te façam o tpc?
   . Já experimentaste compilar outra vez?
   . Já experimentaste desligar e voltar a ligar?
-  . A única solução que vejo para isso é formatar...
\ No newline at end of file
+  . A única solução que vejo para isso é formatar...
+  . Sim, é mesmo um servidor de geeks!
+  . Não... O curso é feito por ti, não por nós!
+  . Acho que estás a merecer um !ja
+  . Se quiseres aprender a programar, podes contar com o nosso apoio!
+  . Se o Google não ajudar, força nisso!
+  , https://youtu.be/a3Z7zEc7AXQ
+  , Já contaste ao patinho?
+  , Ufa, foi por pouco que não te apanhávamos!
+  , Se já viste https://youtu.be/MAlSjtxy5ak talvez aqui tenhas mais sorte!
```

Comment:
Se a frase anterior termina uma vírgula, mudaria o início da frase para minúscula.

Fora isso, lgtm.